redis/types.go

```diff
@@ -0,0 +1,37 @@
+package redis
+
+import (
+	"time"
+
+	"github.com/go-redis/redis"
+)
+
+// Config for redis connection
+// TODO Set all default values
+type Config struct {
+	Host string `yaml:"host" env:"REDIS_HOST" env-description:"Redis host"`
+	Port string `yaml:"port" env:"REDIS_PORT" env-description:"Redis port"`
+	Pass string `env:"REDISPASS" env-description:"Redis user password"`
+}
+
+// repository represent the repository model
+type repository struct {
+	Client redis.Cmdable
```

Comment:
Why do you call this Repository?

Reply:
This is I would like to encapsulate the Client and expose only Get/Set func.

Reply:
Now I see, that from `Connect` you return interface value. 
I would rename the struct and interface to `redis` and `Redis`

Reply:
Okay, but can we change the naming for struct and interface?

Reply:
I will update struct and interface naming as Camille's mention.